jobbergate-agent/tests/jobbergate/test_update.py

```diff
@@ -533,7 +537,7 @@ async def test_fetch_influx_data__success_with_all_None(mocked_influxdb_client:
 @pytest.mark.parametrize(
     "time, host, step, task",
     [
-        tuple(random.randint(1, 100) if i not in combination else None for i in range(4))
+        tuple(1 if i not in combination else None for i in range(4))
```

Comment:
A random parameter made it challenging to work on the project using vscode. I pinned it to 1 for now, since the value doesn't seem relevant on this test. We may revisit it in the future.